styles/styles.css

```diff
@@ -450,6 +450,35 @@ body.resource-center > main > div.section {
   margin-right: auto;
 }
 
+body.resource-center > main > div.section.box-shadow.post-teaser-list-container {
```

Comment:
Why we are not maintaining templates code in templates/template-type/template-type.css file ?
I think it will be hard to maintain as the number of templates increases.

Reply:
There shouldn't be too many templates. I would like to keep them to a minimum